connectors/class-connector-settings.php

```diff
@@ -385,7 +395,24 @@ public function get_serialized_field_label( $option_name, $field_key ) {
 				// Custom Header
 				'header_image'           => esc_html__( 'Header Image', 'stream' ),
 				'header_textcolor'       => esc_html__( 'Text Color', 'stream' ),
+				'header_background_color'=> esc_html__( 'Header and Sidebar Background Color', 'stream' ),
+				// Featured Content
+				'featured_content_layout'=> esc_html__( 'Layout', 'stream' ),
+				// Custom Sidebar
+				'sidebar_textcolor'      => esc_html__( 'Header and Sidebar Text Color', 'stream' ),
+				// Custom Colors
+				'color_scheme'           => esc_html__( 'Color Scheme', 'stream' ),
+				'main_text_color'        => esc_html__( 'Main Text Color', 'stream' ),
+				'secondary_text_color'   => esc_html__( 'Secondary Text Color', 'stream' ),
+				'link_color'             => esc_html__( 'Link Color', 'stream' ),
+				'page_background_color'  => esc_html__( 'Page Background Color', 'stream' ),
 			),
+
+			'twentyeleven_theme_options' => array(
```

Comment:
What do you think of this - just saving a generic "Twenty Eleven Theme Options" setting - instead of going into detail?

I'm not sure it's worth it for the added code complication.

What do you think?